gulpfile.js

```diff
@@ -6,14 +6,14 @@ var
   gulp          = require('gulp'),
   autoprefixer  = require('gulp-autoprefixer'),
   browserSync   = require('browser-sync').create(),
-  copydir       = require('copy-dir');
   sass          = require('gulp-sass'),
   sassGlob      = require('gulp-sass-glob'),
   debug         = require('gulp-debug'),
   modernizr     = require('gulp-modernizr'),
   shell         = require('gulp-shell'),
   sourcemaps    = require('gulp-sourcemaps'),
   prompt        = require('gulp-prompt'),
+  fs            = require('fs'),
```

Comment:
You have a new plugin here "fs" but I don't see it in the package.json file... do we need it in there?